media/admin/com_virtuemart/views/zasilkovna/tmpl/default_carriers.php

```diff
@@ -0,0 +1,29 @@
+<?php defined('_JEXEC') or die('Restricted access');
+    /** @var VirtueMartModelZasilkovna $model */
+    $model = VmModel::getModel('zasilkovna');
+    $updateCarriersToken = $model->getConfig('update_carriers_token');
+
+    $params['task'] = 'updateCarriers';
+    $params['option'] = 'com_virtuemart';
+    $params['view'] = 'plugin';
+    $params['type'] = 'vmshipment';
+    $params['token'] = $updateCarriersToken;
+    $params['name'] = VirtueMartModelZasilkovna::PLG_NAME;
+    $publicUpdateUrl = JUri::root() . 'index.php?' . http_build_query($params);
+    $updateUrl = Juri::base(true) . '/index.php?option=com_virtuemart&view=zasilkovna&task=updateCarriers';
+
+    $lastUpdated = $model->getLastCarriersUpdateTimeFormatted();
+    $carriersCount = $model->getTotalUsableCarriersCount();
+?>
+<div>
```

Comment:
Nešla by použít nějaká třída pro padding-left?

Reply:
Je to i v jiných tabech? Pokud ne, tak bych to nedělal.

Reply:
Toto bych řešil samostatně pro všechny taby najednou. U těch objednávek je to také nalepené.

Reply:
ok